doc/user/shared-content/sql-command-privileges/show-create-type.md

```diff
@@ -0,0 +1 @@
+There are no privileges required to execute this statement.
```

Comment:
Are you sure? I'm no expert on RBAC, but since types are in schemas, I'd naively expect it to need usage privilege on the schema.

Reply:
I see, where can I see an example of one? If it needs usage privileges?

Reply:
This is checked in `rbac.rs`'s `check_plan`: `generate_rbac_requirements` looks at the `Plan::ShowCreate` that `plan_show_create_type` returns. So, I'd expect this to already work for `SHOW CREATE TYPE`, and do the same thing as for other `SHOW CREATE` statements. But please double-check this.

Reply:
You mean "There are no privileges required to execute this statement" is correct? I thought the code would already do the same as for most other SHOW CREATEs, i.e. "`USAGE` privileges on the schema containing the table."

Reply:
Yes, `SHOW CREATE CLUSTER` is special, in that clusters don't live in a schema. But most other things do live in schemas.